src/components/views/dialogs/LogoutDialog.tsx

```diff
@@ -134,6 +134,12 @@ export default class LogoutDialog extends React.Component<IProps, IState> {
                     "Encrypted messages are secured with end-to-end encryption. " +
```

Comment:
> I don't think your changelog entry worked as you would have expected ftr - try just `Notes:` instead of `element-web notes:`
>
> *-- @turt2live, https://github.com/matrix-org/matrix-react-sdk/pull/8741#issuecomment-1145410677*

Weird, I don't think that worked either 🤔

The issue template says to use `element-web notes: <notes>`  which is why I tried that first.

```
<!-- To specify text for the changelog entry (otherwise the PR title will be used):
Notes:

Changes in this project generate changelog entries in element-web by default.
To suppress this:

element-web notes: none

...or to specify different notes:
element-web notes: <notes>
-->
```

Reply:
we'd want the changelog entry in both layers, so would just use `Notes`

Reply:
Ohhh, after your comment, I assume the folowing is how it works? The language in the template is pretty confusing around this so I think I'll give it an update if this is right.

 - `Notes:` is for a changelog entry in https://github.com/matrix-org/matrix-react-sdk/blob/develop/CHANGELOG.md
 - `element-web notes: <notes>` is for a changelog entry in https://github.com/vector-im/element-web/blob/develop/CHANGELOG.md
 
When I use these, should it be updated and reflected in the changelog preview? That's what I was expected.

Reply:
Thanks @turt2live 👍

It works if I simplify the notes ✅. There isn't a way to change the `Contributed by` person but I've just bodged in another one in the notes.